explainer.md

```diff
@@ -0,0 +1,369 @@
+* The ??? (e.g. the bank that issued the payment instrument being used)
+
+> **NOTE**: It is not uncommon for there to be a fourth party: the Payment
+> Service Provider, whom the Merchant may delegate the task of accepting the
+> payment method to. This is usually done by the Merchant either redirecting the
+> User to the Payment Service Provider's website, or by embedding the Payment
+> Service Provider in an iframe. Any following mention of the term Merchant
+> should be taken to mean either a Merchant or a Payment Service Provider;
+> whichever is handling payment.
+
+Traditionally, the Customer only interacts with the Merchant, and the Merchant
+communicates with the ??? via a back-channel protocol. However, ???s are
+increasingly looking to authenticate Customers during online payments. This is
+done for both regulatory reasons (e.g. [SCA] in the European Union) or for
+fraud prevention reasons - either fradulent Customers or fradulent Merchants.
+
+Existing methods of authenticating a Customer during an online payment are
+either high friction (e.g. embedding challenge iframes from the ???) or have
+poor user privacy (e.g. fingerprinting or tracking the user for risk analysis,
+to provide a 'frictionless flow'). The payments industry needs a consistent,
+low friction, strong authentication flow for online payments.
+
+The [Web Authentication (WebAuthn) API][webauthn] makes FIDO-based
+authentication available on the web, which provides a strong, low-friction
+method for a user to prove a pre-established identity with a given site.
+However, WebAuthn is not immediately suitable to solve the payments
+authentication problem, as:
+
+1. It requires the ??? to be present in the transaction (e.g. via an embedded
+   iframe), which increases user friction and lowers conversion rates.
+1. The generated assertion contains no payments-related information, which could
+   be useful to meet e.g. [Dynamic Linking] requirements.
+1. WebAuthn does not allow credential creation in a cross-origin iframe, thus
+   excluding a useful onboarding flow - enrolling a Customer after they have
+   completed a traditional authentication flow (e.g. via SMS OTP).
+
+This proposal attempts to provide an authentication solution for online
+payments that is both as strong and low-friction as WebAuthn, whilst also
+solving these three issues.
+
+### Goals
+
+Find a solution that (in no particular order):
+
+* Has as-strong or stronger authentication than current challenge flows.
+* Is less friction for the Customer than current challenge flows.
+* Is more protective of user-privacy than current frictionless flows.
+* Improves the ability for online payments to meet regulations such as [Dynamic
+  Linking].
+
+> **NOTE**: It is out-of-scope for the specification to **prove** adherence
+> to e.g. Dynamic Linking, but we aim to produce a solution that could be
+> vetted as such.
+
+* Allows for in-flow enrollment of Customers during a traditional challenge flow.
```

Comment:
You might want to mention that enrollment may also happen at other times, so something like:
"Allows customers to enroll a traditional challenge flow (as well as outside of a transaction)."